docs/epiviz/priors_as_data.rst

```diff
@@ -0,0 +1,100 @@
+.. _epiviz-priors-input:
+
+Priors Input
+============
+
+.. _epiviz-priors-as-data:
+
+Priors as Data
+--------------
+
+.. warning::
+
+    This is not implemented.
+
+As described in :ref:`epiviz-posteriors`, a single estimation can create
+integrands for child locations and make point estimates for the value
+and standard error of those child values. These become input for the
+child estimation and are a way to specify prior belief by adding to input
+data. They are added to the bundle as though they came from the bundle.
+If the parent fit did not include random effects, then the input data
+will be from the fit at the parent level.
+
+The `minimum measured CV option <https://bradbell.github.io/dismod_at/doc/option_table.htm#minimum_meas_cv>`_
+in Dismod-AT, represented in the Cascade UI on the Advanced tab,
+will apply to this data as it applies to all data in the bundle.
+
+There is an option in the Cascade user interface
+to limit which of the integrands from the parent to include in the data.
+
+.. attention::
+
+    Does the min cv field in the user interface refer to the data priors,
+    to the fit priors, or to both?
+
+
+Priors on Fit
+-------------
+
+.. warning::
+
+    This is not implemented.
+
+
+As described in :ref:`posteriors-to-priors`, each single estimation can
+use the set of fits, created from simulation draws, in order to create
+priors.
+Because these are priors on the fit, they are priors on only the five
+rates, iota, rho, chi, omega, and pini. We get those priors by looking
+at the five principal integrands (incidence, remission, excess mortality,
+other-cause mortality and initial prevalence) at point values.
+
+However, the priors on fit apply to
+
+ *  primary rates - the value priors, the dage priors, the dtime priors,
+    and the mulstd priors, if present. These values can be calculated from
+    fits or calculate by the Dismod-AT predict function.
```

Comment:
Should we turn DISMOD_AT predict to a hypertext that refers to a description of the predict function.

Reply:
done